src/hooks/use-current-path.ts

```diff
@@ -0,0 +1,36 @@
+import { useRouter } from 'next/router'
+
+interface UseCurrentPathOptions {
+  excludeHash?: boolean
+  excludeSearch?: boolean
+}
+
+/**
+ * Gets the current path similar to Next's router but optionally excludes the
+ * hash and/or search portion of the path. Uses `window.location`.
+ */
+const useCurrentPath = (options: UseCurrentPathOptions = {}): string => {
+  const router = useRouter()
+  const { excludeHash = false, excludeSearch = false } = options
+  const { hash, pathname, search } = new URL(
+    router.asPath,
+    // TODO: replace this with an environment variable soon
+    'http://www.test.com'
```

Comment:
If we're only ever gonna return relative links, it's probably okay to put a dummy URL here...but maybe `https://www.hashicorp.com`? 😄

Reply:
Done 😁